src/berry_mill/sysinfo.py

```diff
@@ -65,14 +61,12 @@ def is_vm() -> bool:
     """
     Detect if the current machine is a VM
     """
-    lshw:str = "/usr/bin/lshw"
-    assert os.path.exists(lshw), f"{lshw} utility is missing"
-
+    assert shutil.which("lshw"), "lshw utility is missing"
```

Comment:
I would rather explicitly look for those paths. I don't want to hit some custom stuff in `/home/luser/bin/lshw` which might be something else or even wrong:

```python
assert bool(list(filter(None, [os.path.exists("{}/lshw".format(p)) for p in ["/usr/bin", "/usr/sbin", "/sbin", "/bin"]]))), f"{lshw} utility is missing"
```
```

Reply:
Good point, yes sure.